format/proto/types.proto

```diff
@@ -6,6 +6,7 @@ enum DataType {
     HOLE = 2;
     TRIMMED = 3;
     PROPOSAL = 4;
+    CHECKPOINT = 5;
```

Comment:
Hmm, no. I think I'll move some of the checkpoint KV dictionary items to the top-level metadata instead.